xml_inscriptions/transcribed/MD.Balt.WAG.G.57.1960.xml

```diff
@@ -133,6 +134,53 @@
       <body>
          <div type="edition">
          <!--The marked up transcription goes here, surrounded by <ab> </ab> elements-->
+            <ab>
+               <lb n="1"/><gap reason="illegible" quantity="1" unit="character"/><g type="characters"/>Ἰάω,
```

Comment:
I'm not entirely sure that this is the right use of `<g>`, though it will translate to what you want it to in the edition itself I think. I'll approve this for now, but I'll continue to think a bit more about how best to represent what's going on here.

Reply:
i agree, i just wasn't sure how else to encode it. i also made sure to link to the seg entry which actually has the specific symbols written out i think. although seg isnt accessible to everybody.